lib/Serialization/SerializedModuleLoader.cpp

```diff
@@ -344,6 +344,7 @@ std::error_code SerializedModuleLoaderBase::openModuleFiles(
 std::error_code SerializedModuleLoader::findModuleFilesInDirectory(
     AccessPathElem ModuleID, StringRef DirPath, StringRef ModuleFilename,
     StringRef ModuleDocFilename, StringRef ModuleSourceInfoFileName,
+    SmallVectorImpl<char> *__ModuleInterfacePath,
```

Comment:
Because the argument is ignored. I didn't see a convention for it anywhere else, so I used the Python convention.

Reply:
The C++ standard [reserves `__` identifiers for the compiler's use](https://timsong-cpp.github.io/cppwp/n4659/lex.name#3.1), so we probably shouldn't use it here. In practice, I think there's no convention for this because nobody particularly sees a need to indicate it in our code bases.